src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -1486,6 +1486,13 @@ void Transformations::MainSnippets() {
                 return output_shape[1].is_dynamic() || output_shape[1].get_length() > 256;
             },
             snippets::pass::TokenizeMLPSeqSnippets);
+        CPU_SET_CALLBACK_ARM(
+            snippetsManager,
+            [&]([[maybe_unused]] const std::shared_ptr<const ov::Node>& n) -> bool {
+                // TODO: Enable TokenizeMLPSeqSnippets on ARM
+                return true;
+            },
+            snippets::pass::TokenizeMLPSeqSnippets);
```

Comment:
Let's better take `ignoreCallback` value into account during `isMlpSeqSupported` computation: this will take less code. And please also move the extended comment there (feel free to reformulate if needed)
```
// Note: Currently, MLPSeqSnippets is enabled only in tests
// TODO: Enable TokenizeMLPSeqSnippets on ARM for all scenarios
```

Reply:
You're right, done